src/_scss/pages/modals/dashboard/_dashboardModal.scss

```diff
@@ -0,0 +1,94 @@
+.dashboard-page__modal {
```

Comment:
The dark gray text color varies slightly within the modal (headings, detail rows, and paragraph are all different) - I would add some overrides to this file to make them consistent.